.circleci/config.yml

```diff
@@ -47,31 +47,79 @@ jobs:
 workflows:
   ci:
     jobs:
+      # Ruby 2.7 releases
+      - bundle_lint_test:
+          name: ruby2-7_rails6-0
+          ruby_version: 2.7.5
+          rails_version: 6.0.4.7
       - bundle_lint_test:
           name: ruby2-7_rails5-2
-          ruby_version: 2.7.0
+          ruby_version: 2.7.5
           rails_version: 5.2.4
+      # Ruby 2.6 releases
+      - bundle_lint_test:
+          name: ruby2-6_rails6-0
+          ruby_version: 2.6.9
+          rails_version: 6.0.4.7
       - bundle_lint_test:
           name: ruby2-6_rails5-2
-          ruby_version: 2.6.5
+          ruby_version: 2.6.9
           rails_version: 5.2.4
+      # Ruby 2.5 releases
+      - bundle_lint_test:
+          name: ruby2-5_rails6.0
+          ruby_version: 2.5.9
+          rails_version: 6.0.4.7
       - bundle_lint_test:
           name: ruby2-5_rails5-2
-          ruby_version: 2.5.7
+          ruby_version: 2.5.9
           rails_version: 5.2.4
+
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - main
+    jobs:
+      # Ruby 2.7 releases
       - bundle_lint_test:
-          name: ruby2-4_rails5-2
-          ruby_version: 2.4.9
-          rails_version: 5.2.4
+          name: ruby2-7_rails6-1
+          ruby_version: 2.7.5
+          rails_version: 6.1.5
```

Comment:
I'm not seeing rails 6.1 being tested in the commit-triggered build.  How about adding it to the on-commit build matrix?  If it doesn't work then I would think it should be removed from both on-commit and nightly builds until it can be resolved.